### PR TITLE
Unittest examples for sampleSel

### PR DESCRIPTION
**Pull request by mdkeehan**
1.  Unit tests to show the "golden path" or a minimum working example of how to call sampleSel with 1 and 2 traits. These test catch the exceptions which are being thrown and hence will need to be modified.
2. A function to clear out the globals so the tests have a known static initial state.
3. Another place that possibly needs yesterdays fix.

### CODE REVIEW

File: src/output/phenotype.jl [hunk 1]
```diff
@@ -19,7 +19,8 @@ function getOurPhenVals(my::Cohort, varRes=0)
     genVals  = getOurGenVals(my,nTraits)
     for (i,animal) = enumerate(my.animalCohort)
         if length(animal.phenVal) == 0
-            animal.phenVal =  animal.genVal + LRes * randn(nTraits)
+	    #for t in 1:nTraits?
```
Comment (mdkeehan) [inline src/output/phenotype.jl:22]: Some multi-trait support needed here

File: src/output/phenotype.jl [hunk 2]
```diff
@@ -45,7 +46,10 @@ function getOurGenVals(my::Cohort,nTraits=0)
             getMyHaps(animal)
             myGenotypes = getMyGenotype(animal)
             #animal.genVal = dot(myGenotypes[common.G.qtl_index],common.G.qtl_effects)
-            animal.genVal = (myGenotypes[common.G.qtl_index]'common.G.qtl_effects)'
```
Comment (mdkeehan) [inline src/output/phenotype.jl:48]: Again some multi-trait support needed here.

File: src/output/phenotype.jl (hunk 1 above)
Comment (reworkhow) [reply src/output/phenotype.jl:22]: Hi Mike,

I added an example here 
https://github.com/reworkhow/XSim.jl/wiki/Multi-trait-Simulation

Hao

**Comment by mdkeehan** [comment]
Hi Hao,

Thanks for the example documentation it really helps.
I have updated my unit tests to follow the example and submitted a pull
request.

Mike

**Pull request merged**